src/components/ToDoWrapper.vue

```diff
@@ -38,4 +42,9 @@
   watch(listItems.value, (newListItems) => {
```

Comment:
you can use a watcheffect instead of a watch here

`watchEffect(() => {
    localStorage.setItem('listOfItems', JSON.stringify(listItems.value));
})`

This will work the same but the code is way more concise